lib/web/files.go

```diff
@@ -79,7 +79,7 @@ func (h *Handler) transferFile(w http.ResponseWriter, r *http.Request, p httprou
 		return nil, trace.Wrap(err)
 	}
 
-	return OK(), nil
+	return nil, nil
```

Comment:
```suggestion
	// we must return nil so that we don't write anything to 
	// the response, which would corrupt the downloaded file	
	return nil, nil
```